tags.go

```diff
@@ -168,20 +169,24 @@ func (s *TagsService) DeleteTag(pid interface{}, tag string, options ...OptionFu
 	return s.client.Do(req, nil)
 }
 
-// CreateReleaseOptions represents the available CreateRelease() options.
+// CreateTagReleaseOptions represents the available CreateRelease() options.
+//
+// Deprecated: This feature was deprecated in GitLab 11.7.
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/tags.html#create-a-new-release
-type CreateReleaseOptions struct {
+type CreateTagReleaseOptions struct {
 	Description *string `url:"description:omitempty" json:"description,omitempty"`
 }
 
 // CreateRelease Add release notes to the existing git tag.
 // If there already exists a release for the given tag, status code 409 is returned.
 //
+// Deprecated: This feature was deprecated in GitLab 11.7.
+//
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/tags.html#create-a-new-release
-func (s *TagsService) CreateRelease(pid interface{}, tag string, opt *CreateReleaseOptions, options ...OptionFunc) (*Release, *Response, error) {
+func (s *TagsService) CreateRelease(pid interface{}, tag string, opt *CreateTagReleaseOptions, options ...OptionFunc) (*TagRelease, *Response, error) {
```

Comment:
Option struct always use the same name as the functions. So in this case the function should be renamed to `CreateTagRelease` (or `CreateReleaseTag` if we like that better).